packages/flutter/test/cupertino/switch_test.dart

```diff
@@ -414,5 +414,27 @@ void main() {
 
     expect(value, isFalse);
   });
+  testWidgets('Switch is opaque when disabled', (WidgetTester tester) async {
+    final Key switchKey = UniqueKey();
```

Comment:
You don't need this key if you're not using it

Reply:
Would you prefer if I did something like this?:
`          child: CupertinoSwitch(
            key: UniqueKey(),
            value: false,
            dragStartBehavior: DragStartBehavior.down,
            onChanged: null,
          ),`
If I remove the key parameter completely it creates a warning

Reply:
A key parameter shouldn't be needed. What warning did it show?

Reply:
'Prefer const with constant constructors.'
Seems to be related to having 'onChanged' set to null. With this code there is a warning:
`
CupertinoSwitch(
            value: false,
            dragStartBehavior: DragStartBehavior.down,
            onChanged: null,
          ),
`
With this code there is not:
`
CupertinoSwitch(
            value: false,
            dragStartBehavior: DragStartBehavior.down,
            onChanged: (bool newValue) {},
          ),
`

Reply:
oh, it just means you can instantiate the CupertinoSwitch (and likely its parent Center) via a compile time constant. (e.g. add a const in front of the Center constructor).

Reply:
I made a change on your branch. The const should be at the highest node of the widget tree declaration possible.